proxy/src/auth/credentials.rs

```diff
@@ -26,6 +26,10 @@ pub struct ClientCredentials {
     // New console API requires SNI info to determine the cluster name.
     // Other Auth backends don't need it.
     pub sni_data: Option<String>,
+
+    // cluster_option is passed as argument from options from url.
+    // To be used to determine cluster name in case sni_data is missing.
+    pub project_name: Option<String>,
```

Comment:
Probably specifying both should be an error. What if they are incoherent?

Reply:
Will add that check. Thanks!